### ISSUE TITLE
Missing length annotations in VkCuLaunchInfoNVX

### ISSUE THREAD

**Issue by expipiplus1** [KhronosGroup/Vulkan-Docs#1548]
Should `pParams` and `pExtras` have length annotations as such:

```xml
            <member len="paramCount">const <type>void</type>* const *    <name>pParams</name></member>
            <member len="extraCount">const <type>void</type>* const *    <name>pExtras</name></member>
```

**Comment by oddhack** [comment]
@dgkoch could you ask Liam and/or Eric to respond to this? I can't assign them until they're members of the project, which they aren't yet.

**Comment by dgkoch** [comment]
cc @liam-middlebrook

**Comment by dgkoch** [comment]
Fixed in 1.2.183 release
